astro.config.mjs

```
@@ -1,46 +1,66 @@
 import icon from "astro-icon";
+import cloudflare from "@astrojs/cloudflare";
 
 // https://astro.build/config
 export default defineConfig({
   site: "https://rasmusj.dk",
+  output: 'server',
+  adapter: cloudflare({
+    imageService: 'passthrough',
+    routes: {
+      extend: {
+        include: [{ pattern: '/static' }], // Route a prerended page to the SSR function for on-demand rendering
```

Biome (1.9.4)

[error] This property value named include is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named include


(lint/suspicious/noDuplicateObjectKeys)